pkg/client/typed/dynamic/client.go

```diff
@@ -177,6 +204,29 @@ func (rc *ResourceClient) Watch(opts v1.ListOptions) (watch.Interface, error) {
 		Watch()
 }
 
+// Get gets the resource with the specified name.
+func (src *SubresourceClient) Get(name string) (*runtime.Unstructured, error) {
```

Comment:
The `name` refers to the resource's name, not the subresources name?  Need to make it clear in the argument name and doc.  `Get` is a little confusing since the name doesn't refer directly to the thing you're getting.